packages/mui-base/src/Menu/Root/useMenuRoot.ts

```diff
@@ -192,8 +195,9 @@ export function useMenuRoot(parameters: UseMenuRootParameters) {
   const typeahead = useTypeahead(floatingRootContext, {
     listRef: itemLabels,
     activeIndex,
+    resetMs: 350,
```

Comment:
@colmtuite  Why so short? For slow typers this seems way too fast to me. macOS' typeahead for Menus is over 500ms (not sure on the exact value, but 750ms seems like a good default).

Reply:
iirc, @michaldudak suggested 350ms? Not sure. I don't think I had a specific preference, just that the original value felt clearly too long.